Windows SqlServer CI Configs sometimes cannot connect to the LocalDB

I suspect the actions runners again...

```
023-06-12T04:17:31.8382858Z Test run for D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\bin\Release\net6.0\Tgstation.Server.Tests.dll (.NETCoreApp,Version=v6.0)
2023-06-12T04:17:31.9673424Z Microsoft (R) Test Execution Command Line Tool Version 17.3.2 (x64)
2023-06-12T04:17:31.9674502Z Copyright (c) Microsoft Corporation.  All rights reserved.
2023-06-12T04:17:31.9743742Z 
2023-06-12T04:17:32.2260096Z Starting test execution, please wait...
2023-06-12T04:17:32.2757858Z A total of 1 test files matched the specified pattern.
2023-06-12T04:18:53.6595386Z 
2023-06-12T04:18:54.3485937Z ##[error]Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.4718657Z 
2023-06-12T04:18:54.5122205Z   Failed TestUpdateProtocolAndDisabledOAuth
2023-06-12T04:18:54.5207622Z   Error Message:
2023-06-12T04:18:54.5215479Z    Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5216632Z   Stack Trace:
2023-06-12T04:18:54.5217828Z      at Tgstation.Server.Tests.Live.TestLiveServer.Initialize(TestContext _) in D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\Live\TestLiveServer.cs:line 160
2023-06-12T04:18:54.5218505Z 
2023-06-12T04:18:54.5218818Z   Standard Output Messages:
2023-06-12T04:18:54.5219681Z  CACHE PREWARMED: https://secure.byond.com/download/build/514/514.1588_byond.zip
2023-06-12T04:18:54.5220425Z  dbug: CachingFileDownloader[0]
2023-06-12T04:18:54.5221434Z        Starting download of https://github.com/tgstation/tgstation-server/releases/download/tgstation-server-v5.11.0/ServerUpdatePackage.zip...
2023-06-12T04:18:54.5222229Z  trce: CachingFileDownloader[0]
2023-06-12T04:18:54.5222890Z        Cached to C:\Users\runneradmin\AppData\Local\Temp\tmp8BD9.tmp
2023-06-12T04:18:54.5223772Z  TEST ERROR INIT: Could not connect to database. Retrying after 3s. Exception: System.Threading.Tasks.TaskCanceledException: A task was canceled.
2023-06-12T04:18:54.5225163Z     at Tgstation.Server.Tests.Live.TestLiveServer.Initialize(TestContext _) in D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\Live\TestLiveServer.cs:line 145
2023-06-12T04:18:54.5225883Z 
2023-06-12T04:18:54.5226034Z 
2023-06-12T04:18:54.5228720Z 
2023-06-12T04:18:54.5234819Z ##[error]Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5237601Z 
2023-06-12T04:18:54.5243483Z   Failed TestUpdateBadVersion
2023-06-12T04:18:54.5244794Z   Error Message:
2023-06-12T04:18:54.5246534Z    Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5248767Z   Stack Trace:
2023-06-12T04:18:54.5249910Z      at Tgstation.Server.Tests.Live.TestLiveServer.Initialize(TestContext _) in D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\Live\TestLiveServer.cs:line 160
2023-06-12T04:18:54.5250653Z 
2023-06-12T04:18:54.5250812Z 
2023-06-12T04:18:54.5253432Z ##[error]Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5255440Z 
2023-06-12T04:18:54.5255834Z   Failed TestOneServerSwarmUpdate
2023-06-12T04:18:54.5270199Z   Error Message:
2023-06-12T04:18:54.5271679Z    Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5272752Z   Stack Trace:
2023-06-12T04:18:54.5273927Z      at Tgstation.Server.Tests.Live.TestLiveServer.Initialize(TestContext _) in D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\Live\TestLiveServer.cs:line 160
2023-06-12T04:18:54.5274575Z 
2023-06-12T04:18:54.5274637Z 
2023-06-12T04:18:54.5276968Z ##[error]Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5278295Z 
2023-06-12T04:18:54.5278574Z   Failed TestCreateServerWithNoArguments
2023-06-12T04:18:54.5279037Z   Error Message:
2023-06-12T04:18:54.5280261Z    Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5281220Z   Stack Trace:
2023-06-12T04:18:54.5282179Z      at Tgstation.Server.Tests.Live.TestLiveServer.Initialize(TestContext _) in D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\Live\TestLiveServer.cs:line 160
2023-06-12T04:18:54.5282794Z 
2023-06-12T04:18:54.5282856Z 
2023-06-12T04:18:54.5285005Z ##[error]Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5286287Z 
2023-06-12T04:18:54.5286559Z   Failed TestSwarmSynchronizationAndUpdates
2023-06-12T04:18:54.5287192Z   Error Message:
2023-06-12T04:18:54.5288408Z    Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5289396Z   Stack Trace:
2023-06-12T04:18:54.5290450Z      at Tgstation.Server.Tests.Live.TestLiveServer.Initialize(TestContext _) in D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\Live\TestLiveServer.cs:line 160
2023-06-12T04:18:54.5291026Z 
2023-06-12T04:18:54.5292347Z 
2023-06-12T04:18:54.5294566Z ##[error]Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5295992Z 
2023-06-12T04:18:54.5296232Z   Failed TestSwarmReconnection
2023-06-12T04:18:54.5296659Z   Error Message:
2023-06-12T04:18:54.5298022Z    Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5298895Z   Stack Trace:
2023-06-12T04:18:54.5299783Z      at Tgstation.Server.Tests.Live.TestLiveServer.Initialize(TestContext _) in D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\Live\TestLiveServer.cs:line 160
2023-06-12T04:18:54.5300366Z 
2023-06-12T04:18:54.5300424Z 
2023-06-12T04:18:54.5302603Z ##[error]Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5304111Z 
2023-06-12T04:18:54.5304358Z   Failed TestStandardTgsOperation
2023-06-12T04:18:54.5304783Z   Error Message:
2023-06-12T04:18:54.5306018Z    Class Initialization method Tgstation.Server.Tests.Live.TestLiveServer.Initialize threw exception. Microsoft.VisualStudio.TestTools.UnitTesting.AssertFailedException: Assert.Fail failed. Could not connect to the test database! Try re-running failed jobs..
2023-06-12T04:18:54.5306952Z   Stack Trace:
2023-06-12T04:18:54.5307949Z      at Tgstation.Server.Tests.Live.TestLiveServer.Initialize(TestContext _) in D:\a\tgstation-server\tgstation-server\tests\Tgstation.Server.Tests\Live\TestLiveServer.cs:line 160
2023-06-12T04:18:54.5308581Z 
2023-06-12T04:19:23.8905989Z 
```

--- a/.github/workflows/ci-suite.yml
+++ b/.github/workflows/ci-suite.yml
@@ -290,6 +290,13 @@ jobs:
       BYOND_MAJOR: 514
       BYOND_MINOR: 1588
     steps:
+    - name: Wait for LocalDB Connection # Do this first because we don't want to find out it's failing later
+      shell: powershell
+      if: ${{ matrix.database-type == 'SqlServer' }}
+      run: |
+        Write-Host "Checking"
+        sqlcmd -l 600 -S "(localdb)\MSSQLLocalDB" -Q "SELECT @@VERSION;"
+
     - name: Setup dotnet
       uses: actions/setup-dotnet@v2
       with:

